User/services/UserService.ts

```diff
@@ -0,0 +1,47 @@
+import  { UserModel }  from '../models/UserModel';
+import { UserDAO } from '../data-access/UserDAO';
+import { User } from '../types/userTypes';
+
+export class UserService {
+
+	public static async getAllUsers(): Promise<UserModel[]> {
+		try {
+			return await UserDAO.getAllUsers();
+		} catch (err) {
+			throw Error(`Something went wong while getting all users: ${err.message}`);
```

Comment:
Better not to show this error message from the end-user, as it can contain some unnecessary data. Like sequelize errors or, in the worst case, some confidential data. Personally, I like to handle the errors in the controller.